main.nf

```diff
@@ -202,35 +203,37 @@ process trim_galore {
 
 process bwa {
 
-    tag "$name"
+    tag "$reads"
 
     module 'bioinfo-tools'
     module 'bwa'
     module 'samtools'
-    module 'BEDTools'
 
-    cpus 4
-    memory { 32.GB * task.attempt }
-    time { 120.h * task.attempt }
+    cpus 8
+    memory { 64.GB * task.attempt }
+    time { 48.h * task.attempt }
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
     maxRetries 3
     maxErrors '-1'
 
     publishDir "${params.outdir}/bwa", mode: 'copy'
 
     input:
-    file (reads:'*') from trimmed_reads
-    set val(name) from name_for_bwa
+    file reads from trimmed_reads
 
     output:
-    file '*.sam' into bwa_sam
+    file '*.bam' into bwa_bam
     stdout into bwa_logs
 
     script:
     """
-    f='$reads';f=(\$f);f=\${f[0]};f=\${f%.gz};f=\${f%.fastq};f=\${f%.fq};f=\${f%.R1_val_1*};f=\${f%.R2_val_2*};f=\${f%_trimmed};f=\${f%_1}
-    prefix=\$f
-    bwa mem -M $index $reads > \${prefix}.sam
+    f='$reads';f=(\$f);f=\${f[0]};f=\${f%.gz};f=\${f%.fastq};f=\${f%.fq};f=\${f%.*1_val_1};f=\${f%_1_val_1};f=\${f%_trimmed}
+
+    bwa \\
+      mem \\
```

Comment:
I know I'm being totally hypocritical here, but as it's such a short command I think it's actually easier to read on single line.. :trollface: